Engine/musmon.c

```diff
@@ -270,7 +270,10 @@ int musmon(CSOUND *csound)
     csound->nspout = csound->ksmps * csound->nchnls;  /* alloc spin & spout */
     csound->nspin = csound->ksmps * csound->inchnls; /* JPff: in preparation */
     csound->spin  = (MYFLT *) csound->Calloc(csound, csound->nspin*sizeof(MYFLT));
-    csound->spraw = (MYFLT *) csound->Calloc(csound, csound->nspout*sizeof(MYFLT));
+    /* VL: 27.1.24 spraw redeployed as thread buffer */
+    printf("threads: %d \n", csound->oparms->numThreads);
```

Comment:
This looks like a debug message for testing? If so, suggest to remove.

Reply:
done.